crates/sui-graphql-rpc/src/types/zklogin_verify_signature.rs

```diff
@@ -36,7 +36,7 @@ pub(crate) enum ZkLoginIntentScope {
 #[derive(SimpleObject, Clone, Debug)]
 pub(crate) struct ZkLoginVerifyResult {
     /// The boolean result of the verification. If true, errors should be empty.
-    success: bool,
+    pub success: bool,
```

Comment:
the typescript cannot find this type definition, i think making this pub should do.

Reply:
This shouldn't be the case -- (i.e. I think it should show up in the GraphQL schema even without the `pub` here) -- but it's also fine to have it!